src/views/webview-app/extension-app-message-constants.ts

```diff
@@ -123,10 +137,12 @@ export type MESSAGE_FROM_WEBVIEW_TO_EXTENSION =
   | OpenConnectionStringInputMessage
   | OpenTrustedLinkMessage
   | RenameConnectionMessage
-  | EditAndConnectConnection;
+  | EditAndConnectConnection
+  | OpenFileChooser;
 
 export type MESSAGE_FROM_EXTENSION_TO_WEBVIEW =
```

Comment:
I know this is from before but I'm curious, is there a reason this type is uppercase snake case?

Reply:
A good point as well :) i don't know the reasoning behind it, but also think it could be lowercase. I will update this!